backend/config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Spree::Core::Engine.add_routes do
   namespace :admin do
+    post '/addresses/create_shipment_address/:shipment_id/', to: "addresses#create_shipment_address", as: :create_shipment_address
```

Comment:
This doesn't look very RESTful. Its assumed that a create occurs because its a POST and I don't think address is needed at the end since `/addresses` is already specified. Maybe:

```ruby
post '/addresses/shipment/:shipment_id', to: "addresses#create_shipment", as: :create_shipment_address
```